### PR TITLE
Converted Video Questions to Markdown (in 08-data-analysis-with-python) (#38991)

### PR DESCRIPTION
**Pull request by JapneetSingh5**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #XXXXX

### CODE REVIEW

File: curriculum/challenges/english/08-data-analysis-with-python/data-analysis-with-python-course/parsing-html-and-saving-data.english.md [hunk 1]
```diff
@@ -15,7 +15,10 @@ videoId: bJaqnTWQmb0
 
 ```yml
 question:
-  text: What Python library has the <code>.read_html()</code> method we can we use for parsing HTML documents and extracting tables?
+  text: What Python library has the 
+  ```python
+  .read_html()
+  ```method we can we use for parsing HTML documents and extracting tables?
```
Comment (Sky020) [inline curriculum/challenges/english/08-data-analysis-with-python/data-analysis-with-python-course/parsing-html-and-saving-data.english.md:21]:
```suggestion
  text: |
     What Python library has the `.read_html()` method we can we use for parsing HTML documents and extracting tables?
```

File: curriculum/challenges/english/08-data-analysis-with-python/data-analysis-with-python-course/python-iteration-and-modules.english.md [hunk 2]
```diff
@@ -17,10 +17,23 @@ videoId: XzosGWLafrY
...
   text: How would you iterate over and print the keys and values of a dictionary named "user"?
   answers:
-    - '<pre>for key in user.items():<br>    print(key)</pre>'
-    - '<pre>for key, value in user.all():<br>    print(key, value)<br>    print(value)'
-    - '<pre>for key, value in user.items():<br>    print(key, value)</pre>'
-    - '<pre>for key, value in user<br>    print(key, value)</pre>'
+    - ```python
+      for key in user.items():
+          print(key)
+       ```
```
Comment (Sky020) [inline curriculum/challenges/english/08-data-analysis-with-python/data-analysis-with-python-course/python-iteration-and-modules.english.md:23]:
```suggestion
      ```
```

File: curriculum/challenges/english/08-data-analysis-with-python/data-analysis-with-python-course/reading-data-from-databases.english.md [hunk 3]
```diff
@@ -17,9 +17,9 @@ videoId: MtgXS1MofRw
 question:
   text: What method does a <code>Cursor</code> instance have and what does it allow?
   answers:
-    - The <code>Cursor</code> instance has a <code>.run()</code> method which allows you to run SQL queries.
-    - The <code>Cursor</code> instance has a <code>.select()</code> method which allows you to select records.
-    - The <code>Cursor</code> instance has an <code>.execute()</code> method which will receive SQL parameters to run against the database.
+    - The ```python Cursor ```instance has a```python .run()``` method which allows you to run SQL queries.
+    - The ```python Cursor ```instance has a```python .select()``` method which allows you to select records.
+    - The ```python Cursor ```instance has an```python .execute()``` method which will receive SQL parameters to run against the database.
```
Comment (Sky020) [inline curriculum/challenges/english/08-data-analysis-with-python/data-analysis-with-python-course/reading-data-from-databases.english.md:22]:
```suggestion
    - |
      The `Cursor` instance has a `.run()` method which allows you to run SQL queries.
    - |
      The `Cursor` instance has a `.select()` method which allows you to select records.
    - |
      The `Cursor` instance has an `.execute()` method which will receive SQL parameters to run against the database.
```

File: curriculum/challenges/english/08-data-analysis-with-python/numpy/basics-of-numpy.english.md [hunk 4]
```diff
@@ -15,11 +15,23 @@ videoId: f9QrZrKQMLI
 
 ```yml
 question:
-  text: 'What will the following code print?:<pre>b = np.array([[1.0,2.0,3.0],[3.0,4.0,5.0]])<br>print(b)</pre>'
+  text: |
+    'What will the following code print?:
```
Comment (Sky020) [inline curriculum/challenges/english/08-data-analysis-with-python/numpy/basics-of-numpy.english.md:19]:
```suggestion
    What will the following code print?:
```

File: curriculum/challenges/english/08-data-analysis-with-python/data-analysis-with-python-course/reading-data-csv-and-txt.english.md [hunk 5]
```diff
@@ -17,10 +17,22 @@ videoId: ViGEv0zOzUk
...
+      ```
+    - ```python
+      import pandas as pd
+      pd.read_csv("data.csv")
+      ```
+    - ```python
+      import pandas as pd<br>
+      df = pd.csv_reader("data.csv")
+      ```
   solution: 2
```
Comment (Sky020) [inline curriculum/challenges/english/08-data-analysis-with-python/data-analysis-with-python-course/reading-data-csv-and-txt.english.md:36]:
```suggestion
    - |
       ```python
       import pandas as pd
       df = pd.csv("data.csv")
       ```
    - |
       ```python
       import pandas as pd<br>
       df = pd.read_csv("data.csv")
       ```
    - |
       ```python
       import pandas as pd
       pd.read_csv("data.csv")
       ```
    - |
       ```python
       import pandas as pd<br>
       df = pd.csv_reader("data.csv")
       ```
  solution: 2
```

File: curriculum/challenges/english/08-data-analysis-with-python/numpy/basics-of-numpy.english.md [hunk 6]
```diff
@@ -15,11 +15,23 @@ videoId: f9QrZrKQMLI
...
+    - |
+      `[[1.0 2.0 3.0]
+       [3.0 4.0 5.0]]`
+    - |
+      `[[1. 2. 3.]
+       [3. 4. 5.]]`
+    - |
+      `[[1. 3.]
+       [2. 4.]
+       [3. 5.]`
```
Comment (Sky020) [inline curriculum/challenges/english/08-data-analysis-with-python/numpy/basics-of-numpy.english.md:34]:
```suggestion
    - |
       ```python
       [[1.0 2.0 3.0]
       [3.0 4.0 5.0]]
       ```
    - |
       ```python
       [[1. 2. 3.]
       [3. 4. 5.]]
       ```
    - |
       ```python
       [[1. 3.]
       [2. 4.]
       [3. 5.]
       ```
```

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]

**Comment by JapneetSingh5** [comment]
Thankyou for the suggestions, @Sky020 , goes a long way in improving my abilities and my love for open source!

**Comment by JapneetSingh5** [comment]
opened a new pull request, https://github.com/freeCodeCamp/freeCodeCamp/pull/39023

**Comment by Sky020** [comment]
@JapneetSingh5 , you can keep everything in the same PR. It makes keeping track of changes much easier, as well as build fails. If we see a build fail, we can either fix what is causing the failure, or, if the build freezes (sometimes happens) we can run them again from the same PR.

File: curriculum/challenges/english/08-data-analysis-with-python/data-analysis-with-python-course/python-iteration-and-modules.english.md [hunk 7]
```diff
@@ -17,10 +17,23 @@ videoId: XzosGWLafrY
...
+          print(value)
+      ```
+    - ```python
+      for key, value in user.items():
+          print(key, value)
+      ```
+    - ```python
+      for key, value in user
+          print(key, value)
+      ```
```
Comment (Sky020) [inline curriculum/challenges/english/08-data-analysis-with-python/data-analysis-with-python-course/python-iteration-and-modules.english.md:36]:
```suggestion
    - |
       ```python
       for key in user.items():
           print(key)
       ```
    - |
       ```python
       for key, value in user.all():
           print(key, value)
           print(value)
       ```
    - |
       ```python
       for key, value in user.items():
           print(key, value)
       ```
    - |
       ```python
       for key, value in user
           print(key, value)
       ```
```

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]

**Review by moT01 — COMMENTED** [review]
LGTM @JapneetSingh5 🎉 This looks like a good PR that would have been merged. However, a PR was previously merged that changed all these to use the correct format already. I don't believe this is necessary anymore, so I am going to close it - if anyone disagrees, let me know and we can re-open it. Thanks and happy coding 🎉

**Pull request closed without merging**